nss-day-cohort-51/nss-day-cohort-51.github.io
# mm-feature-description-70 updated description information

# Description

## Type of change

Updating description section

# Testing Instructions
View description section to see updates

ADDITION:
Added a "More Details" button on the Event Card.
Added styling to fall in-line with existing buttons.

# Checklist:

- [] I have commented my code, particularly in hard-to-understand areas
- [] My changes generate no new warnings or errors
